packages/shared/src/authz/policies/project/tokens/canReadEmbedTokens.spec.ts

```diff
@@ -0,0 +1,63 @@
+import cryptoRandomString from 'crypto-random-string'
+import { canReadEmbedTokensPolicy } from './canReadEmbedTokens.js'
+import { Roles } from '../../../../core/constants.js'
+import { assert, describe, expect, it } from 'vitest'
+import { getProjectFake, getWorkspaceFake } from '../../../../tests/fakes.js'
+import { parseFeatureFlags } from '../../../../environment/index.js'
+import { ProjectNoAccessError } from '../../../domain/authErrors.js'
+
+const buildCanReadEmbedTokens = (
+  overrides?: Partial<Parameters<typeof canReadEmbedTokensPolicy>[0]>
```

Comment:
`OverridesOf<typeof canReadEmbedTokensPolicy>`

Reply:
love it